2nd-Seminar/2nd-Seminar/ViewController.swift

```diff
@@ -0,0 +1,787 @@
+import UIKit
+import SnapKit
```

Comment:
UIKit이나 Foundation 같은 표준 라이브러리는 외부 라이브러리와 구분하기 위해 한 칸 띄어주는 것이 좋습니다.
```suggestion
import UIKit

import SnapKit
```